src/pages/PointerMotivation.tsx

```diff
@@ -2,21 +2,22 @@ import { FC } from 'react';
 import AppWrapper from '../components/AppWrapper';
 import NavButtons from '../components/NavButtons';
 import { HeaderSections } from '../types/globalTypes';
-import '../styles/PointerMotivation.scss';
 
 const PointerMotivation: FC = () => {
   return (
     <div>
       <AppWrapper section={HeaderSections.TITLE_SECTION}>
-        <div id="lesson">
-          <h1 id="title">Why Pointers?</h1>
-          <p id="description">
+        <div className="page-wrapper">
+          <h1>Why Pointers?</h1>
+          <p>
             But first, why do we even need pointers? Even though they&apos;re
             annoying to learn, pointers are actually really useful. Click
             through to see the many different ways they can be helpful:
           </p>
-          {/* insert slide show and prev/next buttons */}
-          <p id="description">
+          <div className="diagram">
+            {/* insert slide show and prev/next buttons */}
```

Comment:
Change comment to just insert slide show, since the prev/next buttons are already in.